### PR TITLE
dropping support for python3.3 and some minor setup.py cleaning

### LINKED ISSUE #281: Drop support for python 3.3
It has reached end-of-life
https://www.python.org/downloads/release/python-337/ on 2017-09-29

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #281

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,9 +13,6 @@ matrix:
         - os: linux
           python: 3.2
           env: TOXENV=py32 VENVVER="<14.0.0" PIPVER="<8.0.0" STVER="<30.0.0"
```
Comment (hugovk) [inline .travis.yml]: Drop Python 3.2 as well.

**Comment by hugovk** [comment]
It'd be good to add something like this to setup.py to make sure pip installs the correct version for the user's Python version.

```python
python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
```

File: .travis.yml (hunk 1 above)
Comment (grzanka) [reply .travis.yml]: As being said here: https://github.com/DataMedSci/mcpartools/pull/100#discussion_r154989417

We still have (unfortunately) some users running Debian 7 which has Python 3.2 installed. I'd delay dropping python3.2 until May 2018 when Debian 7 will be EOL.
I'll do it gladly !

File: setup.py [hunk 2]
```diff
@@ -140,21 +147,20 @@ def get_cmdclass():
         # that you indicate whether you support Python 2, Python 3 or both.
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3.2',
-        'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
```
Comment (ant6) [inline setup.py]: Py 3.7 missing?

Comment (ant6) [reply setup.py]: It is included in tox.ini/tests.

**Pull request merged**